git_machete/cmd.py

```diff
@@ -3108,27 +3171,30 @@ def usage(c=None):
         """[1:])))
 
 
-def short_usage():
+def short_usage() -> None:
     print(fmt("<b>Usage: git machete [--debug] [-h] [-v|--verbose] [--version] <command> [command-specific options] [command-specific argument]</b>"))
 
 
-def version():
+def version() -> None:
     print(f"git-machete version {__version__}")
 
 
-def main():
+def main() -> None:
     launch(sys.argv[1:])
 
 
-def launch(orig_args):
+definition_file_path: str = ""
+
+
+def launch(orig_args: List[str]) -> None:
     if sys.version_info.major == 2:
         sys.stderr.write("Python 2.x is no longer supported. Please switch to Python 3.6 or higher.\n")
         sys.exit(1)
     elif sys.version_info.major == 3 and sys.version_info.minor < 6:
         sys.stderr.write("Python 3.x below 3.6 is no longer supported. Please switch to Python 3.6 or higher.\n")
```

Comment:
Unrel: maybe instead of `3.x below 3.6` simply include the actual version from `sys.version_info` for clarity